cf/polyfills.js

```diff
@@ -81,19 +89,23 @@ export const fs = {
 }
 
 export const net = {
-  isIP: x => RegExp.prototype.test.call(IPv4Reg, x) ? 4 : RegExp.prototype.test.call(IPv6Reg, x) ? 6 : 0,
+  isIP: (x) =>
+    RegExp.prototype.test.call(IPv4Reg, x)
+      ? 4
+      : RegExp.prototype.test.call(IPv6Reg, x)
+        ? 6
+        : 0,
   Socket
 }
 
 export { setImmediate, clearImmediate }
 
 export const tls = {
-  connect(x) {
-    const tcp = x.socket
+  connect({ socket: tcp, servername }) {
```

Comment:
Destructuring here 🤔 
Not sure if you want to keep this, maybe a merge side-effect. Feel free to keep it or not - this code works (I've added some tests files down this PR)